examples/md-flexible/src/Generator.h

```diff
@@ -83,7 +83,7 @@ void Generator::cubeGauss(autopas::AutoPas<Particle, ParticleCell> &autopas, con
   dummyParticle.setTypeId(object.getTypeId());
   autopasTools::generators::GaussianGenerator::fillWithParticles(
       autopas, object.getBoxMin(), object.getBoxMax(), object.getParticlesTotal(), dummyParticle,
-      object.getDistributionMean(), object.getDistributionMean());
+      object.getDistributionMean(), object.getDistributionStdDev());
```

Comment:
This change makes `md-flexible.test-sim` (defined in `examples/md-flexible/CMakeLists.txt:59`) fail, as there are really really fast particles.
Quick fix would be to change the stddeviation of that test...

Reply:
we probably should not use a randomly generated scenario with deltaT != 0 as a test then ;)